utils/src/main/java/org/jboss/windup/util/FurnaceCompositeClassLoader.java

```diff
@@ -83,11 +83,12 @@ public URL getResource(String name)
         ClassLoader contextLoader = Thread.currentThread().getContextClassLoader();
         if (contextLoader != null)
         {
-            return contextLoader.getResources(name);
+            result.addAll(Collections.list(contextLoader.getResources(name)));
```

Comment:
result should be a LinkedHashSet. It doesn't seem valid to have two identical resources in it. Same goes for the list of classloaders in this class